feelslikemonday/app/src/main/java/com/example/feelslikemonday/ui/following/FolloweeBookAdapter.java

```diff
@@ -0,0 +1,54 @@
+package com.example.feelslikemonday.ui.following;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.TextView;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import com.example.feelslikemonday.R;
+import com.example.feelslikemonday.model.followeeMoodEvent;
+import java.util.ArrayList;
+
+/*
+ *This class acts as a array adapter for the followee's on the following page
+*/
+
+public class FolloweeBookAdapter extends ArrayAdapter<followeeMoodEvent> {
+
+    private Context context;
+    private int mResource;
+
+    public FolloweeBookAdapter(Context context, int resource, ArrayList<followeeMoodEvent> objects) {
+        super(context, resource, objects);
+        this.context = context;
+        mResource = resource;
+    }
+
+    @NonNull
+    @Override
+    public View getView(int position, @Nullable View convertView, @NonNull ViewGroup parent) {
+
+        String var0_fUser = getItem(position).getUsername().toString();
```

Comment:
This naming should not use lowercases but rather be lowerCamerCased. We should try to avoid naming our variables varX_xxxx

Reply:
done